mumctl/src/main.rs

```diff
@@ -525,17 +521,17 @@ fn match_server_config(matches: &clap::ArgMatches<'_>, config: &mut mumlib::conf
                     server
                         .port
                         .map(|s| format!("port: {}\n", s))
-                        .unwrap_or("".to_string()),
+                        .unwrap_or_else(|| "".to_string()),
```

Comment:
I guess this is fine. I don't really like the way this looks, but a lazy generator is better I guess.